src/quantuminspire/qiskit/backend_qx.py

```diff
@@ -30,12 +30,12 @@
 from qiskit.result.models import ExperimentResult, ExperimentResultData
 from qiskit.validation.base import Obj
 
-from quantuminspire.exceptions import QisKitBackendError
 from quantuminspire.qiskit.circuit_parser import CircuitToString
 from quantuminspire.qiskit.qi_job import QIJob
-from quantuminspire.version import __version__ as quantum_inspire_version
-from quantuminspire.job import QuantumInspireJob
 from quantuminspire.api import QuantumInspireAPI
+from quantuminspire.exceptions import QisKitBackendError
+from quantuminspire.job import QuantumInspireJob
+from quantuminspire.version import __version__ as quantum_inspire_version
 
 
 class QuantumInspireBackend(BaseBackend):
```

Comment:
There are still a lot of typehints int the __init__ docstring

Reply:
The docstring types in init is the description of the used configuration. As the docstring descriptions in api.py, this is not replaced by type hinting because these structures/classes are not ours.